src/bin/main.rs

```diff
@@ -53,7 +53,7 @@ fn core() -> Result<i32, String> {
                 },
             )?;
 
-            println!("A new Bonnie configuration file has been initialized at [PATH]!");
+            println!("A new Bonnie configuration file has been initialized at ./bonnie.toml!");
```

Comment:
You should put a `match` here to account for the `BONNIE_CONF` environment variable, which may change that path.

Reply:
I think this is the last thing left!

Reply:
Currently `bonnie -i` does not pay attention to the `BONNIE_CONF` environment variable:
``` rust
if matches!(template, Some(_)) && fs::metadata(template.as_ref().unwrap()).is_ok() {
    let template_path = template.unwrap();
    // We have a valid template file
    let contents = fs::read_to_string(&template_path);
    let contents = match contents {
        Ok(contents) => contents,
        Err(_) => return Err(format!("An error occurred while attempting to read the given template file '{}'. Please make sure the file exists and you have the permissions necessary to read from it.", &template_path))
    };
    output = fs::write("./bonnie.toml", contents);
} // [other branches...]
```

Reply:
Ah. That's a severe oversight then on my part! In that case we can resolve this and merge, and I'll fix that before the next version.